### PR TITLE
Rank the hidden gems based on rating (descending order)

### PR DESCRIPTION
**Pull request by helenatran**
This PR sorts the hidden gems by rating (descending order), which will be used for the ranking.

### CODE REVIEW

**Review by sehhajdkg — APPROVED** [review]
Look Good. I couldn't find any issues :)

File: src/main/java/com/google/sps/data/Places.java [hunk 1]
```diff
@@ -143,4 +146,21 @@
     }
     return stream;
   }
+
+  /**
+   * This function returns the hidden gems ranked by rating (descending order)
+   * @param hiddenGems                      This takes as parameter the set of hiddenGems to be sorted. 
+   * @return ArrayList<PlacesSearchResult>  This returns an arrayList of the hidden gems ranked by rating.
+   */
+  public static ArrayList<PlacesSearchResult> getRankedHiddenGems(Set<PlacesSearchResult> hiddenGems) {
```
Comment (lorikeet-s) [inline src/main/java/com/google/sps/data/Places.java:155]: Unless you're relying on something specific about arraylist probably better to return this as a List

**Review by lorikeet-s — CHANGES_REQUESTED** [review]
one minor comment, otherwise looks good

**Review by jamila-sabaz — APPROVED** [review]

File: src/main/java/com/google/sps/data/Places.java (hunk 1 above)
Comment (g-wittie) [reply src/main/java/com/google/sps/data/Places.java:155]: Yep, all List implementations are ordered.

Comment (helenatran) [reply src/main/java/com/google/sps/data/Places.java]: I have changed it to a List

**Review by lorikeet-s — APPROVED** [review]

**Pull request merged**